@navikt/core/react/src/date/Time/time.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import { Meta } from "@storybook/react";
+import React from "react";
+import { Time } from "./Time";
+
+export default {
+  title: "ds-react/Time",
+  component: Time,
+} as Meta;
+
+export const Default = () => {
+  return <Time date={new Date()} />;
```

Comment:
The example dates should be static, or else snapshots will be different all the time.

Some edge cases that it might be nice to include are values <10 and hours>12.